cmd/user-agent/src/pages/chapi/Login.vue

```diff
@@ -60,11 +62,22 @@ SPDX-License-Identifier: Apache-2.0
             if (wuser) {
                 this.registered = true
                 this.statusMsg = `Wallet already registered for ${wuser.id}`
+            } else {
```

Comment:
what's the motive behind this logic?

Reply:
all these logics in login vue is gone now, and in future this gonna be replaced by hub-auth page redirect.
Please rebase.

Reply:
Rebased. Originally I had that logic in order to pre-populate the username field based on the previously used username, and I also had it generating a random username for new users. These were just nice-to-haves while it was in development, and would make username collisions harder... but seeing as how it's going to be moved soon it's probably not that important at the moment. I removed that extra logic.